package.json

```diff
@@ -110,29 +111,32 @@
 		"node": ">=12"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.16.5",
-		"@babel/eslint-parser": "^7.16.5",
+		"@babel/core": "^7.17.2",
+		"@babel/eslint-parser": "^7.17.0",
 		"@babel/plugin-syntax-top-level-await": "^7.14.5",
```

Comment:
Are these deps still required? You *at least* removed `@babel/eslint-parser` from the eslint config below.

Reply:
Still having an interesting experience with mjs and eslint sadly, but I might not need this anymore

Reply:
if you use @typescript-eslint/parser you need to define `extraFileExtensions: ['.mjs']`. That might just be all you need. That's all that is done for `.mjs` support for Sapphire anyway and we have maaany mjs scripts.

See: https://github.com/sapphiredev/utilities/blob/9e8ac5661e9a05299cba5bd58eb69d435676359f/packages/eslint-config/src/index.ts#L14